test/cli.js

```diff
@@ -59,15 +59,15 @@ describe('cli', function() {
   });
 
   it('should output stylelint warnings', function(done) {
```

Comment:
I mean a nested `describe` in `test/cli.js` with the `importRoot` relevant tests.
The point testing imports indirectly i.e. by tweaking other tests to cover some use cases it is not optimal.